nilearn/load_confounds/data/missing_desc-confounds_regressors.tsv

```diff
@@ -0,0 +1,2 @@
+csf	csf_derivative1	csf_power2	csf_derivative1_power2	white_matter	white_matter_derivative1	white_matter_derivative1_power2	white_matter_power2	global_signal	global_signal_derivative1	global_signal_power2	global_signal_derivative1_power2	std_dvars	dvars	framewise_displacement	t_comp_cor_00	t_comp_cor_01	t_comp_cor_02	t_comp_cor_03	t_comp_cor_04	t_comp_cor_05	a_comp_cor_00	a_comp_cor_01	a_comp_cor_02	a_comp_cor_03	a_comp_cor_04	a_comp_cor_05	a_comp_cor_06	a_comp_cor_07	a_comp_cor_08	a_comp_cor_09	a_comp_cor_10	a_comp_cor_11	a_comp_cor_12	a_comp_cor_13	a_comp_cor_14	a_comp_cor_15	a_comp_cor_16	a_comp_cor_17	a_comp_cor_18	a_comp_cor_19	a_comp_cor_20	a_comp_cor_21	a_comp_cor_22	a_comp_cor_23	a_comp_cor_24	a_comp_cor_25	a_comp_cor_26	a_comp_cor_27	a_comp_cor_28	a_comp_cor_29	a_comp_cor_30	a_comp_cor_31	a_comp_cor_32	a_comp_cor_33	a_comp_cor_34	a_comp_cor_35	a_comp_cor_36	a_comp_cor_37	a_comp_cor_38	a_comp_cor_39	a_comp_cor_40	a_comp_cor_41	a_comp_cor_42	a_comp_cor_43	a_comp_cor_44	a_comp_cor_45	a_comp_cor_46	a_comp_cor_47	a_comp_cor_48	a_comp_cor_49	a_comp_cor_50	a_comp_cor_51	a_comp_cor_52	a_comp_cor_53	a_comp_cor_54	a_comp_cor_55	a_comp_cor_56	a_comp_cor_57	a_comp_cor_58	a_comp_cor_59	a_comp_cor_60	a_comp_cor_61	a_comp_cor_62	a_comp_cor_63	a_comp_cor_64	a_comp_cor_65	a_comp_cor_66	a_comp_cor_67	a_comp_cor_68	a_comp_cor_69	a_comp_cor_70	a_comp_cor_71	a_comp_cor_72	a_comp_cor_73	a_comp_cor_74	a_comp_cor_75	a_comp_cor_76	a_comp_cor_77	a_comp_cor_78	a_comp_cor_79	a_comp_cor_80	a_comp_cor_81	a_comp_cor_82	a_comp_cor_83	a_comp_cor_84	a_comp_cor_85	a_comp_cor_86	a_comp_cor_87	a_comp_cor_88	a_comp_cor_89	a_comp_cor_90	a_comp_cor_91	a_comp_cor_92	a_comp_cor_93	a_comp_cor_94	a_comp_cor_95	a_comp_cor_96	a_comp_cor_97	a_comp_cor_98	a_comp_cor_99	a_comp_cor_100	a_comp_cor_101	a_comp_cor_102	a_comp_cor_103	a_comp_cor_104	a_comp_cor_105	a_comp_cor_106	a_comp_cor_107	a_comp_cor_108	a_comp_cor_109	a_comp_cor_110	a_comp_cor_111	a_comp_cor_112	a_comp_cor_113	a_comp_cor_114	a_comp_cor_115	a_comp_cor_116	a_comp_cor_117	a_comp_cor_118	a_comp_cor_119	a_comp_cor_120	a_comp_cor_121	a_comp_cor_122	a_comp_cor_123	a_comp_cor_124	a_comp_cor_125	trans_x	trans_x_power2	trans_x_derivative1_power2	trans_y_derivative1	trans_y_derivative1_power2	trans_y_power2	trans_z	trans_z_derivative1	trans_z_power2	trans_z_derivative1_power2	rot_x	rot_x_derivative1	rot_x_derivative1_power2	rot_x_power2	rot_y	rot_y_derivative1	rot_y_derivative1_power2	rot_y_power2	rot_z	rot_z_derivative1	rot_z_derivative1_power2
```

Comment:
What are the empty files for ?

Reply:
For this specific one, it is a place holder for testing the compcor noise component required files. If the associated json is missing, even when the confound timeseries tsv exist, it will raise an error. There's no point to put a real file here so it's only got one line.

The empty imaging files are place holder to test the file grabbing function. When a functional imaging derivative is given, the `load` method look for the associated confound timeseries.

Reply:
If it is for testing, you maye generate them on-the-fly when runing the tests ?